v3/integration/genTestCerts/genTestCerts.go

```diff
@@ -0,0 +1,314 @@
+package main
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"os/exec"
+	"strings"
+	"sync"
+	"time"
+)
+
+// Generates a CA, an intermediate, and a leaf certificate and prints their
+// OpenSSL textual output to stdout.
+func main() {
+	ca, err := NewTrustAnchor()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
small nit: it might be useful to define a small function that panics on a non-nil err and then use it throughout to reduce the duplication in `main` somewhat. 

```go
panicOnErr := func(err error) {
  if err != nil {
    panic(err)
  }
}
ca, err := NewTrustAnchor()
panicOnErr(err)
fmted, err := OpenSSLFormatCertificate(ca)
panicOnErr(err)
...
```